dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -543,3 +596,32 @@ describe('Clean up all test data', () => {
   });
 });
 
+describe('Verify Operational Panels Table', () =>{
+  it('Add Dummy Data to Operational Panel and verify Table Column, Pagination and Rows Data ', () => {
+  var i = 0;
+  for (i = 0; i < 12 ; i++) { 
```

Comment:
`for (let i = 0; i < 12; i++)` should work as well.

Reply:
What is 12 here? Why did you choose 12 instead of say 15 or 20? This can be easily explained if you have 12 as a value of a defined variable. For example: `const numberOfPanels = 12` and using `numberOfPanels` in the for loop like `for (let i = 0; i < numberOfPanels; i++)`

Reply:
done as per comment